### PR TITLE
Week 12 project by Tiina Liukkonen & Marianne Ardin

### PR DESCRIPTION
**Pull request by MarianneArdin**
Link to my deployed project: https://tiina-marianne-movies.netlify.app/

### CODE REVIEW

File: code/src/Components/MovieDetails.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                .then(data => setMovieDetails(data))
+                .catch(error => console.error(error))
+                .finally(() => setLoading(false))
+        }
+
+        fetchMovieDetails()
+    }, [movieId])    
+    
+
+    console.log("moviedetails", moviedetails)
```
Comment (Mimmisen) [inline code/src/Components/MovieDetails.js:36]: Here you forgot to remove a console log :)

File: code/src/Components/MovieDetails.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                fontSizeDesktop="2rem"
+                letterSpacing="5px"
+                >
+                <span className="button-arrow">
+                <FaChevronLeft />
+                </span>
+                    Back
+                </Button>
+                <div className="moviedetails-wrapper"key={moviedetails.id}>
+                    <img className="movie-poster" src={`https://image.tmdb.org/t/p/w342${moviedetails.poster_path}`} alt=""/>
```
Comment (Mimmisen) [inline code/src/Components/MovieDetails.js:65]: here there is a empty `alt=""` :)

File: code/src/App.js [hunk 3]
```diff
@@ -1,9 +1,24 @@
...
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <BrowserRouter>
+      <div>
+        <Routes>
+            <Route path='/' element={<MovieList />} />
+            <Route path='/moviedetails/:movieId' element={<MovieDetails />} />
+            <Route path='*' element={<Error />} />
```
Comment (Mimmisen) [inline code/src/App.js:17]: This is how I and Lovisa did the handling of errors, check it out if you are interested :) 
   ```
     <Route path="/404" element={<NotFond />} />
        <Route path="*" element={<Navigate to="/404" replace />} />
```

**Review by LovisaBrorson — COMMENTED** [review]
Good job! It's a very nice page. Big plus for the loader. Everything works perfectly and you reached bule and read level, bravo! 

//Mimmi & Lovisa

File: code/src/App.js (hunk 3 above)
Comment (MarianneArdin) [reply code/src/App.js:17]: Hi! Thanks for your comment. With the path of '*' the react router will render all other paths to the error page. It works the same as your solution.

> This is how I and Lovisa did the handling of errors, check it out if you are interested :)
> 
> ```
>   <Route path="/404" element={<NotFond />} />
>      <Route path="*" element={<Navigate to="/404" replace />} />
> ```

**Pull request closed without merging**